### ISSUE TITLE
Lots of "gen unknown" on new x570p AMD pcie4 machine

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/growlight#36]
This is on killermike. We've got one SATA SSD and two SATA spinning disks.

╭──────[ahci-1 (256Gbps to Southbridge, 6Gbps (2%) demanded)]──────────────[-]─╮
│        sda┌─────────────────────────────────────────────────────────────────┐│
│✔ removable│me1 ext4 at / 1233333333333 ext4 at /home (199.96G) 33333333333em││
│31°  no i/o└┤Samsung SSD 840  6B0Q 256.06G  512B gpt   50025385a02983fa SAT3├┘│
╰─────[PCI Express device 0000:07.00.0 (x16, gen unknown)]─────────────────────╯

╭──────[ahci-0 (256Gbps to Southbridge, 12Gbps (4%) demanded)]─────────────[-]─╮
│        sdc┌─⇗⇨⇨⇨empty space─────────────────────────────────────────────────┐│
│✔ removable│me11111111111111111 btrfs at /media/store (12T) 1111111111111111m││
│38°  no i/o└┤ST12000NM0007-2A SN02  12.00T 4096B gpt   5000c500b408015d SAT3├┘│
│        sdb┌─────────────────────────────────────────────────────────────────┐│
│✔ removable│me1111111111111 zfs_member “pbrstreetgang” (12T) 11111111111119em││
│37°  no i/o└┤ST12000NM0007-2A SN02  12.00T 4096B gpt   5000c500b4b8415d SAT3├┘│
╰─────[PCI Express device 0000:08.00.0 (x16, gen unknown)]─────────────────────╯

**Comment by dankamongmen** [comment]
```
╭──────[ahci-0 (256Gbps to Southbridge, 6Gbps (2%) demanded)]──────────────[-]─╮
│        sdb┌─⇗⇨⇨⇨empty space─────────────────────────────────────────────────┐│
│✔ removable│me1 ext4 at / 1233333333333 ext4 at /home (199.96G) 33333333333em││
│31°   308Ki└┤Samsung SSD 840  6B0Q 256.06G  512B gpt   50025385a02983fa SAT3├┘│
╰─────[PCI Express device 0000:08.00.0 (x16, gen 4.0)]─────────────────────────╯

╭──────[ahci-1 (256Gbps to Southbridge, 6Gbps (2%) demanded)]──────────────[-]─╮
│        sda┌─────────────────────────────────────────────────────────────────┐│
│✔ removable│me11111111111111111 btrfs at /media/store (12T) 1111111111111111m││
│35°  no i/o└┤ST12000NM0007-2A SN02  12.00T 4096B gpt   5000c500b4b8415d SAT3├┘│
╰─────[PCI Express device 0000:07.00.0 (x16, gen 4.0)]─────────────────────────╯
```

**Comment by dankamongmen** [comment]
fixed!

**Comment by dankamongmen** [comment]
Fixed in 1.1.1-pre.
